### DISCUSSION TITLE
May I ask when version 6.0 will be released?

### DISCUSSION THREAD

**Discussion by ysl-ccc** [fabricjs/fabric.js#9385] [Q&A]
I mainly want to use the erase module in react+ts, but the erase module does not support ts.

**Comment by ShaMan123** [comment]
We work hard to make it happen
I built the eraser so believe that I want it in.
We have 100 open prs, a lot of work.
Most of them will wait after v6.
We thought v6 would have come out by now. Sorry.
Upvote eraser!

**Reply by ysl-ccc** [reply]
Ok, thank you. Thank you for your hard work
